src/services/utils/gql/types/parseNative.ts

```diff
@@ -35,10 +33,6 @@ export function inferTypeByTarget(target: Function, key?: string) {
     return returnType;
   }
 
-  const targetField = (target as any)[key];
-
-  if (targetField && typeof targetField === 'function') {
```

Comment:
I've realized its not needed to do this. We check it before by checking `returntype`. It's present only if field is function so we dont have to check it again. It also solved problem of calling getter fields